frontend/src/pages/NicknamePage/index.tsx

```diff
@@ -20,8 +20,6 @@ const NicknamePage = ({ gameCode }: Props) => {
   const [error, setError] = useState("");
   const [playerId, setPlayerId] = usePlayerIdState("");
   const [token, setToken] = useTokenState("");
-
-  const memoryHistory = useHistory();
   const browserHistory = useContext(BrowserHistoryContext);
 
   useEffect(() => {
```

Comment:
I'm encountering the following error on the NicknamePage component:

<img width="572" alt="Screen Shot 2021-05-07 at 3 12 24 AM" src="https://user-images.githubusercontent.com/32445605/117403756-cca2b780-af5c-11eb-9808-fd63a54bb672.png">

It appears that this webpage offers a potential fix by wrapping the call for this `async` with a check for if the component is mounted